### DISCUSSION TITLE
Meghibásodott, megsemmisült, elveszett felhőalapú pénztárgép helyettesítése

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#31] [Kérdések és válaszok]
A rendelettervezet II. 5. § (2) a) pontja szerinti értelmezés arra utal, hogy felhőalapú pénztárgép meghibásodás, megsemmisülés esetén csak hardveralapúval helyettesíthető?

Ha a felhőalapú a pénztárgép mobiltelefonra telepített alkalmazásként működik, akkor be kell-e jelenteni a NAV-nak, ha pl. ellopják a telefont?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
